### ISSUE TITLE
Selecting an aggregate without selections causes a null dereference

### ISSUE THREAD

**Issue by schveiguy** [burner/graphqld#21]
Example

```
{
   item {
      subfield
   }
}
```

Where subfield requires selections causes segfault.

**Comment by burner** [comment]
thanks

**Comment by schveiguy** [comment]
another example query:

```graphql
{allEmployees
{
  info
  {
    forename
    addressInstance
  }
}}
```

**Comment by burner** [comment]
I can reproduce it now, thanks

**Comment by burner** [comment]
should be fixed in 0.10

**Comment by burner** [comment]
should be fixed in 0.10.3
